Airgap Support VM custom SSL Certs (#2846)
spectrocloud/librarium: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md

    sudo --login
    ```

> 19. If you have custom SSL certificates you want to apply to the image and pack registry, and the Spectro Cloud
>     Repository, copy the custom SSL certificates, in base64 PEM format, to the airgap support VM. The custom
>     certificates must be placed in the **/opt/spectro/ssl** folder. Include the following files:

Reviewer: ```suggestion
19. If you want to apply custom SSL certificates to the image and pack registry and the Spectro Cloud
    Repository, copy the certificates in the base64 PEM format to the **/opt/spectro/ssl** folder in your airgap support VM. Include the following files:
```

    - **server.crt**
    - **server.key**

>     If you don't provide custom certificate then the airgap setup process will generate a self-signed certificate for
>     you.

Reviewer: ```suggestion
    If you do not provide a custom SSL certificate, the airgap setup process will generate a self-signed certificate for
    you.
```

    :::warning

>     The custom SSL certificates must be in base64 PEM format. If you have custom SSL certificates in a different format,
>     convert them to base64 PEM format before copying them to the support VM. The airgap setup process also expects the
>     files to be named **server.crt** and **server.key**.

Reviewer: ```suggestion
    The custom SSL certificates must be in the base64 PEM format. If you have custom SSL certificates in a different format,
    convert them to the base64 PEM format before copying them to the support VM. The airgap setup process also expects the
    files to be named **server.crt** and **server.key**.
```
